### PR TITLE
fix #13

### LINKED ISSUE #13: Pattern facade for simplicity interface
What do you think about this pattern for lib?
For users is simpler to know about single class that gathering all allowed methods for lib.
 like here https://github.com/andrew-svirin/cfonb-php/blob/master/src/CfonbParser.php
https://refactoring.guru/fr/design-patterns/facade

Also this approach simplifies access to lib, because it is enough to add to container single lib.
image

### PR DESCRIPTION
**Pull request by fezfez**
@andrew-svirin is it ok for you ?

### CODE REVIEW

**Comment by andrew-svirin** [comment]
@fezfez Yes. Now is almost good! But in readme is old access to readers. And Readers can be internal services.

File: src/CfonbReader.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private $cfonb240Reader;
+
+    public function __construct(Cfonb120Reader $cfonb120Reader = null, Cfonb240Reader $cfonb240Reader = null)
+    {
+        $this->cfonb120Reader = null === $cfonb120Reader ? new Cfonb120Reader() : $cfonb120Reader;
+        $this->cfonb240Reader = null === $cfonb240Reader ? new Cfonb240Reader() : $cfonb240Reader;
+    }
+
+    /** @return Statement[] */
+    public function parserCfonb120(string $content): array
```
Comment (guillaume-sainthillier) [inline src/CfonbReader.php:34]: Could you rename method name to `parseCfonb120`?

**Review by guillaume-sainthillier — COMMENTED** [review]
I agree with the Facade pattern as long we can continue to create readers directly in our projects :)

File: src/CfonbReader.php (hunk 1 above)
Comment (fezfez) [reply src/CfonbReader.php]: done

**Comment by fezfez** [comment]
> @fezfez Yes. Now is almost good! But in readme is old access to readers. And Readers can be internal services.

Cool :)

According to @guillaume-sainthillier and myself, we wont drop support for Readers usage without facade.
In this ways, there will  be no BC.

**Comment by fezfez** [comment]
@guillaume-sainthillier ready to merge + tag 2.1.0 :)

**Pull request merged**